schema.prisma

```diff
@@ -618,3 +621,20 @@ enum BankAccountStatus {
 
   @@map("bank_account_status")
 }
+
+enum CampaignFileRole {
+  background
+  coordinator
+  campaignPhoto
+  invoice
+  document
+
+  @@map("role")
```

Comment:
This should be `campaign_file_role` as it will be very very confusing when the column name is `role` and column type is `role`

Should be 

```suggestion
  @@map("campaign_file_role")
```